### PR TITLE
Add --host_swiftcopt

### PR DESCRIPTION
**Pull request by keith**
This allows you to pass arguments to swiftc host targets.

### CODE REVIEW

File: src/test/java/com/google/devtools/build/lib/rules/apple/SwiftConfigurationTest.java [hunk 1]
```diff
@@ -58,4 +59,38 @@ public void testSkylarkApi() throws Exception {
...
+        "   copts = ctx.fragments.swift.copts()",
+        "   return struct(",
+        "      copts=copts,",
+        "   )",
+        "swift_binary = rule(",
+        "   implementation = swift_binary_impl,",
+        "   fragments = ['swift']",
+        ")");
+
+    scratch.file("examples/swift_skylark/a.m");
```
[the remark this answers is not in the document]
Comment (keith) [reply src/test/java/com/google/devtools/build/lib/rules/apple/SwiftConfigurationTest.java]: You're right, removed

**Comment by keith** [comment]
Thanks for reviewing @allevato, does anyone else need to review before we can merge this?

**Comment by allevato** [comment]
I don't have permissions to merge into bazelbuild/bazel, so can you also sanity check this please @lberki?

**Comment by lberki** [comment]
Looks good to me!

**Comment by lberki** [comment]
I sent out an internal code review to merge this change.

**Comment by keith** [comment]
Thanks!

**Pull request closed without merging**